[2024-05-12T11:48:53.667+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-12T11:48:53.697+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_dag.transform_data scheduled__2024-04-29T00:00:00+00:00 [queued]>
[2024-05-12T11:48:53.710+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_dag.transform_data scheduled__2024-04-29T00:00:00+00:00 [queued]>
[2024-05-12T11:48:53.711+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-05-12T11:48:53.737+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): transform_data> on 2024-04-29 00:00:00+00:00
[2024-05-12T11:48:53.744+0000] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'my_dag', 'transform_data', 'scheduled__2024-04-29T00:00:00+00:00', '--job-id', '37', '--raw', '--subdir', 'DAGS_FOLDER/script.py', '--cfg-path', '/tmp/tmphh6q2789']
[2024-05-12T11:48:53.746+0000] {standard_task_runner.py:91} INFO - Job 37: Subtask transform_data
[2024-05-12T11:48:53.748+0000] {logging_mixin.py:188} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=668) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-05-12T11:48:53.749+0000] {standard_task_runner.py:63} INFO - Started process 681 to run task
[2024-05-12T11:48:53.804+0000] {task_command.py:426} INFO - Running <TaskInstance: my_dag.transform_data scheduled__2024-04-29T00:00:00+00:00 [running]> on host 06d3d031dcd2
[2024-05-12T11:48:53.900+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='my_dag' AIRFLOW_CTX_TASK_ID='transform_data' AIRFLOW_CTX_EXECUTION_DATE='2024-04-29T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-04-29T00:00:00+00:00'
[2024-05-12T11:48:53.901+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-12T11:48:53.922+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-12T11:48:53.923+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-12T11:48:53.933+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=my_dag, task_id=transform_data, run_id=scheduled__2024-04-29T00:00:00+00:00, execution_date=20240429T000000, start_date=20240512T114853, end_date=20240512T114853
[2024-05-12T11:48:53.963+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-12T11:48:53.997+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-05-12T11:48:54.005+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
